torch/distributed/run.py

```diff
@@ -548,6 +548,16 @@ def get_args_parser() -> ArgumentParser:
         help="Tee std streams into a log file and also to console (see --redirects for format).",
     )
 
+    parser.add_argument(
+        "--filter-ranks",
```

Comment:
Overall looks good. Technically this is filtering using local ranks?

Reply:
Yeah, I can change the wording. Although, I don't think we can see non-local ranks logs of a different node?

Reply:
thats a good point, bc anyone trying to use this command from a higher level would need to map from global ranks to local ranks when configuring this arg.  I'm not sure if we intend anyone besides local devgpu users to use this feature.  but if we do, we might want to address global rank usability down the road.